### PR TITLE
[MLMC] Update ExaQUte API

### PR DESCRIPTION
**Pull request by riccardotosi**
**Description**
This PR updates the ExaQUte API, as specified in [T. Karasek, S. Böhm, B. Keith, R. Amela, R. Badia, R. Rossi and C. Soriano, D4.3 Benchmarking report as tested on the available infrastructure, Open Access Repository of the ExaQUte project: Deliverables (2021)](https://www.scipedia.com/public/table_Soriano_2019f).

Nightly tests are being checked [here](https://github.com/KratosMultiphysics/Kratos/actions/runs/725369231).

Please mark the PR with appropriate tags: 
- Api Breaker, External library

**Changelog**
Please summarize the changes in one list to generate the changelog:
E.g.
- Update ExaQUte API
- Update MLMC applicationto the new ExaQUte API
- Update xmc to the new ExaQUte API

### CODE REVIEW

File: applications/MultilevelMonteCarloApplication/python_scripts/mlmc_utilities.py [hunk 1]
```diff
@@ -1,43 +1,57 @@
-from __future__ import absolute_import, division # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
-
-# Importing the Kratos Library
-import KratosMultiphysics
-from KratosMultiphysics.MultilevelMonteCarloApplication.tools import ParametersWrapper
-
-# Import packages
+# Import Python libraries
```
Comment (marcnunezc) [inline applications/MultilevelMonteCarloApplication/python_scripts/mlmc_utilities.py:1]: Is this file (`mlmc_utilities.py`) still in use with XMC? Just curious

**Review by marcnunezc — DISMISSED** [review]
Ok if nightly works fine

File: applications/MultilevelMonteCarloApplication/python_scripts/mlmc_utilities.py (hunk 1 above)
Comment (riccardotosi) [reply applications/MultilevelMonteCarloApplication/python_scripts/mlmc_utilities.py:1]: This is the original MC-MLMC synchronous and asynchronous algorithms. It is not used in XMC

**Review by riccardotosi — COMMENTED** [review]
We should add ExaQUte local API license.

**Comment by philbucher** [comment]
@riccardotosi I recommend to merge master
after #8571 the builds should be much faster

**Comment by philbucher** [comment]
> @riccardotosi I recommend to merge master
> after #8571 the builds should be much faster

I mean bcs you run the nightly manual

**Comment by riccardotosi** [comment]
@KratosMultiphysics/uncertainty-quantification can you approve? We can merge this.

**Review by roigcarlo — APPROVED** [review]

**Pull request merged**